apps/entity-service/src/entities/processors/association-processor.ts

```diff
@@ -21,19 +26,26 @@ export abstract class AssociationProcessor<M extends UuidModel<M>, D extends Ass
    */
   static buildSimpleProcessor<M extends UuidModel<M>, D extends AssociationDto<D>>(
     dtoClass: Type<D>,
-    associationGetter: (entity: EntityModel, entityLaravelType: string) => Promise<M[]>
+    associationGetter: (entity: EntityModel, entityLaravelType: string, query?: MediaQueryDto) => Promise<M[]>,
+    totalGetter?: (entity: EntityModel, entityLaravelType: string, query?: MediaQueryDto) => Promise<number>
```

Comment:
Since media is the only one supporting pagination for now and it's now got its own subclass of `AssociationProcessor`, it might be a good idea to roll back this complication from the "simple" processor. I think it's reasonable for an association that needs pagination support to have its own processor subclass. The simpler processor was really just a way to make it easy to provide support for very simple associations.

Reply:
removed